<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpSettingDlg</class>
 <widget class="QWidget" name="IpSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{border:2px solid black;};</string>
     </property>
     <property name="title">
      <string>IP设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>IP2：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="IP2lineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>IP4：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="IP3lineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>IP1：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>IP3：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="IP4lineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="IP1lineEdit"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="saveBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
